client/src/dataset/components/DatasetParams.tsx

```diff
@@ -19,12 +19,23 @@ const renderParams = (params: any) => {
     return Object.keys(params).map((key: string, idx: number) => {
         // Only show parameters, not additional info
         if(!isAdditionalInfo(key)) {
-            return (
-                <Table.Row key={idx}>
-                    <Table.Cell>{key}</Table.Cell>
-                    <Table.Cell>{renderParamValue(params[key])}</Table.Cell>
-                </Table.Row>
-            );
+            if(typeof params[key] === 'object' && !(params[key] instanceof Array)) {
+                return Object.keys(params[key]).map((objKey: string, keyIdx: number) => {
+                    return (
+                        <Table.Row key={keyIdx}>
+                            <Table.Cell>{objKey}</Table.Cell>
+                            <Table.Cell>{renderParamValue(params[key][objKey])}</Table.Cell>
+                        </Table.Row>
+                    );
+                });
+            } else {
+                return (
+                    <Table.Row key={idx}>
+                        <Table.Cell>{key}</Table.Cell>
+                        <Table.Cell>{renderParamValue(params[key])}</Table.Cell>
+                    </Table.Row>
+                );
+            }
```

Comment:
As the two cases are structurally the same, maybe the generation of the `Row` can be extracted into its own function